traffic_ops/app/lib/Schema/Result/DeliveryServiceInfoForCdnList.pm

```diff
@@ -0,0 +1,103 @@
+
+
+#
+# Copyright 2015 Comcast Cable Communications Management, LLC
```

Comment:
The Schema/Result files are all generated from `db/admin.pl reverse_schema`.   Use that and replace this file.

Since it's generated,  no copyright header is required,  but for all other source files,   please use the Apache license and not Comcast.